settings.py

```diff
@@ -1,85 +1,77 @@
-# Django settings for donations project.
-
-try:
-    import local
-except ImportError:
-    import example_local as local
-from django.core.urlresolvers import reverse
-import itertools
 import os
 
-BASE_DIR = os.path.dirname(__file__)
-
-DEBUG = local.DEBUG
+from django.core.urlresolvers import reverse
+from django.utils.translation import gettext_lazy as _
 
-ALLOWED_HOSTS = local.ALLOWED_HOSTS
+import environ
 
-DOMAIN = local.DOMAIN
+env = environ.Env()
+environ.Env.read_env()
 
-ADMINS = local.ADMINS
+BASE_DIR = os.path.dirname(__file__)
 
-MANAGERS = ADMINS
+DEBUG = env.bool('DEBUG', default=False)
+ALLOWED_HOSTS = env.list('ALLOWED_HOSTS', default=[])
 
-DATABASES = local.DATABASES
+DOMAIN = env.str('DOMAIN', default='')
 
-PAYPAL_TEST = local.PAYPAL_TEST
+PAYPAL_TEST = env.bool('PAYPAL_TEST', default=True)
 
-# Append slash seems to be the way to go overall
-APPEND_SLASH = True
+# If a database URL is set, set up the db. Else, assume development
+# and use sqlite.
+if env.str('DATABASE_URL', default=''):
+    DATABASES = {
+        'default': env.db(),
+    }
+else:
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.sqlite3',
+            'NAME': 'db/testdb',
+        },
+    }
 
-# Local time zone for this installation. Choices can be found here:
-# http://en.wikipedia.org/wiki/List_of_tz_zones_by_name
-# although not all choices may be available on all operating systems.
-# In a Windows environment this must be set to your system time zone.
-TIME_ZONE = local.TIME_ZONE
+TIME_ZONE = env.str('TIME_ZONE', default='America/New_York')
 
-# Language code for this installation. All choices can be found here:
-# http://www.i18nguy.com/unicode/language-identifiers.html
 LANGUAGE_CODE = 'en-us'
 
+# For the sites framework. We should get rid of this someday.
```

Comment:
The default configuration shouldn't have this at all, since most sites won't actually have Sites installed. If they do, they should already know if they need the setting or not.

Reply:
The reason I left it in is because the setting was already set and the sites framework is loaded. Can we remove this without causing breakage for users?

Reply:
As far as I can tell, removing the setting doesn't break anything in the tracker, because there's no actual direct references to it. It's possible we're using it indirectly, but I don't think so. We can leave it in for now and I'll write a chore to figure out if we need it at all.